lib/archethic/metrics/metric_client.ex

```diff
@@ -0,0 +1,51 @@
+defmodule ArchEthic.Metrics.MetricClient do
```

Comment:
I think you can merge this module and `ArchEthic.Metrics.NetworkPoller` into `ArchEthicWeb.Metrics.Poller`